dallinger/data.py

```diff
@@ -201,11 +201,14 @@ def export(id, local=False, scrub_pii=False):
         file.write(id)
 
     print("Zipping up the package...")
-    shutil.make_archive(
-        os.path.join("data", id + "-data"),
-        "zip",
-        os.path.join("data", id)
-    )
+
+    dst = os.path.join("data", id + "-data.zip")
+    src = os.path.join("data", id)
+    zf = ZipFile(dst, "w", ZIP_DEFLATED, allowZip64=True)
+    for root, dirs, files in os.walk(src):
+        for file in files:
+            zf.write(os.path.join(root, file))
+    zf.close()
```

Comment:
You can use ZipFile as a context manager to avoid needing to explicitly call .close():

    with ZipFile(dst, 'w', ZIP_DEFLATED, allowZip64=True) as zf:
        # write files

If you put this in a separate function then you can write a test for that function and avoid decreasing coverage.